jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -446,7 +452,7 @@ def _uncertainty(result):
                     raw_result = analysis.line_flux(wave_spec)
                     # When flux is equivalent to Jy, lineflux result should be shown in W/m2
                     if flux_unit.is_equivalent(u.Unit('W/m2/m'/u.sr)):
-                        final_unit = u.Unit('W/m2/sr')
+                        final_unit = self._unit_ignore_warnings('W/m2/sr')
```

Comment:
We have full control of how the unit string is written here. Why not just change the string so it stops emitting warning?
```suggestion
                        final_unit = u.Unit('W / (m2 sr)')
```

Reply:
Huh, the original ticket said that warnings were being thrown from our use of "W / m2 / m", and I totally didn't even consider that the warning I was actually seeing was for "W / m2 / sr" which, as you said, can be rewritten. Let me think about this a little more, your point that we don't even need the filtering might be correct.

Reply:
W / m2 / sr may still be the preferred format for any user-facing representation of the unit... but if it's all internal, then that would definitely be the "easier" fix.

Reply:
p.s. We should not encourage users to write unit in a format that emits astropy warnings. 😉

Reply:
I was going to make the same point Kyle, this is really about how it's displayed to the user, and it may be that the desired format is the one with more slashes that naturally parses to "Watts per square meter per steradian". @PatrickOgle can you chime in on this?

Reply:
`WARNING: UnitsWarning: 'W/m2/sr' contains multiple slashes, which is discouraged by the FITS standard` 🤷